minidump-processor/src/processor.rs

```diff
@@ -381,7 +382,23 @@ where
             thread_context.as_deref()
         };
 
-        let stack_memory = thread.stack_memory(&memory_list);
+        let mut stack_memory = thread.stack_memory(&memory_list);
```

Comment:
Nit: I think this variable should be called something like `referenced_memory`, since it might end up *not* being the stack memory.